app/frontend/action_plans/action_plan_statistics.component.js

```diff
@@ -0,0 +1,30 @@
+export default ({
+  statistics
+}) => (
+  <ul>
+    <li>
+      <span className="number">{statistics.related_proposals_count}</span>
```

Comment:
In order to do a map the `statistics` should be an array which is not a bad idea (now it's an object) but I can iterate an object using ES6 `for-of` loop.